web/api/router.go

```diff
@@ -67,6 +69,18 @@ func WebRouterPrefixOption(prefix string) func(*Router) error {
 	}
 }
 
+// AuthorizationOption is an option to set casbin enforcer for authorization.
+func AuthorizationOption(params []interface{}) func(r *Router) error {
```

Comment:
do we need it?

Reply:
Because do not want to make authorization mandatory, don't we?